backend/btrixcloud/models.py

```diff
@@ -1327,6 +1482,9 @@ class CollOut(BaseMongoModel):
 
     allowPublicDownload: bool = True
 
+    pages: List[PageOut] = []
```

Comment:
Maybe `seedPages` would be more accurate?

Reply:
Could be featuredPages, though maybe that's something we'd add later.

Reply:
renamed to seedPages for now, perhaps we'll add a separate featurePages later, and allow switching between them, etc...